.vscode/tasks.json

```diff
@@ -9,61 +9,61 @@
     {
       "label": "worker",
       "type": "shell",
-      "command": "inv worker",
+      "command": "invoke worker",
```

Comment:
There are a bunch of `invoke` commands in this file that need to be updated to the new format